app/views/shared/_header.html.haml

```diff
@@ -6,6 +6,7 @@
       %li= link_to 'Videos', videos_path
       %li= link_to 'My Queue', my_queue_path
       %li= link_to 'People', people_path
+      %li= link_to 'Invite a friend', invite_path
     = form_tag search_videos_path, method: :post, class: 'col-md-5 navbar-form' do
       .form-group
         = text_field_tag 'term', params[:term] || '', placeholder: 'Search for videos here', class: 'form-control'
```

Comment:
You shouldn't need `|| ''`, if `params[:term]` is `nil`, the value will be blank.